cms/templates/js/course-outline.underscore

```diff
@@ -116,6 +119,13 @@ if (xblockInfo.get('graded')) {
                         <% } %>
                     </span>
                 </p>
+            <% } else if (xblockInfo.getExamScore() !=null) { %>
+                <p>
+                    <span class="status-release-value">
```

Comment:
Ideally this would actually be named something like status-entrance-exam. In an attempt to minimize impact this change would have to the rest of the code, I would have status-entrance-exam and status-exam appended to existing class styles. Inheritance could work as follows:

.status-release, .status-exam {
     .status-release-value, .status-exam-value {
      }
}